packages/graphql/src/schemaTypes/objectTypes/gql-CurrentProject.ts

```diff
@@ -238,6 +238,38 @@ export const CurrentProject = objectType({
       description: 'If the browser is open or not',
       resolve: (source, args, ctx) => ctx.coreData.app.browserStatus,
     })
+
+    t.field('relevantRunSpecs', {
+      description: 'Returns the spec counts for the current and next runs',
+      type: objectType({
+        name: 'CurrentProjectRelevantRunSpecs',
+        definition (t) {
+          t.field('current', {
+            type: RelevantRunSpecs,
+            description: 'Spec counts for the current run',
+          })
+
+          t.field('next', {
+            type: RelevantRunSpecs,
+            description: 'Spec counts for the next run',
+          })
+        },
+      }),
+      resolve: async (source, args, ctx) => {
+        return ctx.relevantRunSpecs.specs
+      },
+    })
+
+    //TODO: Finish implementing resolver for this field to supply runner with tests names to use for filtering
+    t.list.nonNull.string('testsForReviewBySpec', {
+      description: '',
+      args: {
+        specTitle: stringArg(),
+      },
+      resolve: (source, args, ctx) => {
+        return ['foo/bar']
+      },
+    })
```

Comment:
Is this a follow-on task? M2?

Reply:
Will be addressed here: https://github.com/cypress-io/cypress/issues/25487